inc/saihostintf.h

```diff
@@ -556,6 +556,9 @@ typedef enum _sai_hostif_packet_attr
     /** Ingress LAG [sai_object_id_t] (for receive-only) */
     SAI_HOSTIF_PACKET_INGRESS_LAG,
 
+    /** Ingress VLAN [sai_vlan_id_t] (for receive-only) */
+    SAI_HOSTIF_PACKET_INGRESS_VLAN,
```

Comment:
can you clarify the ingress vlan? What is this ingress vlan? Is it internal vlan tag added by the ASIC? What if the packet is untagged packet, or already tagged packet?